docs/5.x/archiving-behavior-specification.md

```diff
@@ -321,7 +321,7 @@ The values of these options are also cached so the tracker doesn't have to query
 
 core:archive can detect when the core archiving process fails, since it will not be able to parse the output of a climulti:request
 command or the archiving curl request. When this occurs, the invalidation being processed's status is reset to 0, so it will be picked up
-again. BUT we also take note of the idinvalidation and skip it for the rest of the current core:archive run. In case the problem
+again. BUT we also take note of the invalidation and skip it for the rest of the current core:archive run. In case the problem
```

Comment:
I guess with `idinvalidation` a certain id of the invalidation was meant (which is used e.g. in the database as identifier). So it might have been correct to refer to the id here, but should have been set in quotes or similar.

Reply:
Ah, good spot, that makes sense. Will revert this change and force-push an update.

Reply:
Reverted and added explanation in brackets after it